gems/activerecord/7.0/activerecord-7.0.rbs

```diff
@@ -4,4 +4,8 @@ module ActiveRecord
       def primary_abstract_class: () -> void
     end
   end
+
+  module Enum
+    def enum: (?(String | Symbol) name, ?(Hash[(Symbol | String), untyped] | Array[Symbol | String]) values, **untyped options) -> void
```

Comment:
ActiveRecord::Enum#enum method in 7.0 version
https://github.com/rails/rails/blob/c15ee6e7b5065db3c46afa1f025b8e45f443604e/activerecord/lib/active_record/enum.rb#L167